site/en/workflows.md

```diff
@@ -598,16 +598,24 @@ your image twice on page level, and have no large images, you can probably skip
       <td>ocrd-skimage-binarize</td>
       <td>
 	  <p><code>
-{"level-of-operation": "region"}
+{"level-of-operation": "region",
+"input_feature_filter": "binarized",
+  "output_feature_added": "binarized",
+  "command": "scribo-cli sauvola-ms-split '@INFILE' '@OUTFILE' --enable-negate-output"
+  }
       </code></p>	
 	  </td>
       <td>
 	  </td>
+      <td><code>
+	  ocrd-preprocess-image -I OCR-D-IMG -O OCR-D-PREP -p '{"level-of-operation": "region","output_feature_added": "binarized","command": "scribo-cli sauvola-ms-split '@INFILE' '@OUTFILE' --enable-negate-output"}'
+	  </code></td>
+	  </td>
```

Comment:
Looks like these lines are in the wrong place. They should be in the `tr` that belongs to `ocrd-preprocess-image` below (which still uses `ocrd-tesserocr-binarize` as example).